web-api/src/persistence/postgres/messages/getMessageById.ts

```diff
@@ -0,0 +1,19 @@
+import { Message } from '@shared/business/entities/Message';
+import { getDbReader } from '@web-api/database';
+import { transformNullToUndefined } from '../utils/transformNullToUndefined';
+
+export const getMessageById = async ({
+  messageId,
+}: {
+  messageId: string;
+}): Promise<Message> => {
+  const message = await getDbReader(reader =>
+    reader
+      .selectFrom('message')
+      .where('messageId', '=', messageId)
+      .selectAll()
+      .executeTakeFirst(),
+  );
+
+  return new Message(transformNullToUndefined({ ...message })).validate();
```

Comment:
I think our persistence methods should be returning a RawMessage as opposed to a Message, as I would expect to pull data out of a database and not an entity. What do you both think?

I also think we would want to avoid validating when fetching from our DB.